src/Data/HugeNum.purs

```diff
@@ -73,6 +74,14 @@ timesSign Plus Plus = Plus
 timesSign Minus Minus = Plus
 timesSign _ _ = Minus
 
+instance isForeignHugeNum :: IsForeign HugeNum where
+  read f = do
+    n <- readNumber f
+    pure (fromNumber n)
+
+instance asForeignHugeNum :: AsForeign HugeNum where
+  write v = write (toNumber v)
```

Comment:
`HugeNum`s are not foreign (in that they're not a type that "comes from JavaScript"), so I don't think we should provide these instances. Also seems like these instances would only work for `HugeNum`s in the range of `Number`, which seems a bit redundant.

We could provide `EncodeJson` / `DecodeJson` instances for argonaut if the idea is to have these for serialisation.